app/javascript/pages/account/create-account.tsx

```diff
@@ -0,0 +1,152 @@
+import Layout from "../../layouts/Layout"
+import NameFieldset from "./components/NameFieldset"
+import { useForm } from "react-hook-form"
+import DOBFieldset from "./components/DOBFieldset"
+import EmailFieldset from "./components/EmailFieldset"
+import PasswordFieldset from "./components/PasswordFieldset"
+import { FormHeader, FormSection } from "../../util/accountUtil"
+import "./styles/account.scss"
+
+interface CreateAccountProps {
+  assetPaths: unknown
+}
+
+const CreateAccountFormSection = ({ children }: { children: React.ReactNode }) => {
+  return <FormSection className="py-4 md:py-8 md:px-10">{children}</FormSection>
+}
+
+const NameSection = () => {
+  const {
+    register,
+    formState: { errors },
+  } = useForm({ mode: "all" })
+
+  return (
+    <CreateAccountFormSection>
+      <NameFieldset
+        register={register}
+        errors={errors}
+        defaultFirstName={null}
+        defaultMiddleName={null}
+        defaultLastName={null}
+      />
+    </CreateAccountFormSection>
+  )
+}
+
+const DateOfBirthSection = () => {
+  const {
+    register,
+    formState: { errors },
+    watch,
+  } = useForm({ mode: "all" })
+
+  return (
+    <CreateAccountFormSection>
+      <DOBFieldset
+        required
+        defaultDOB={null}
+        register={register}
+        error={errors.dob}
+        watch={watch}
+        note={
+          <>
+            <div className="pb-2">{t("createAccount.dobExample")}</div>
+            <div>{t("createAccount.dobNote")}</div>
+          </>
+        }
+      />
+    </CreateAccountFormSection>
+  )
+}
+
+const EmailSection = () => {
+  const {
+    register,
+    formState: { errors },
+  } = useForm({ mode: "all" })
+
+  return (
+    <CreateAccountFormSection>
+      <EmailFieldset
+        register={register}
+        errors={errors}
+        defaultEmail={null}
+        note={t("createAccount.emailNote")}
+      />
+    </CreateAccountFormSection>
+  )
+}
+
+const PasswordSection = () => {
+  const {
+    register,
+    formState: { errors },
+  } = useForm({ mode: "all" })
+
+  return (
+    <CreateAccountFormSection>
+      <PasswordFieldset register={register} errors={errors} />
+      <div className="flex justify-center pt-4">
+        <Button styleType={AppearanceStyleType.primary} type="submit">
+          {t("label.createAccount")}
+        </Button>
+      </div>
+    </CreateAccountFormSection>
+  )
+}
+
+const signInRedirect = () => {
+  console.log("sign in ")
+}
+
+const CreateAccountFooter = () => {
+  return (
+    <Card.Section
+      divider="flush"
+      className="flex justify-center pt-8 text-center w-full flex-col items-center"
+    >
+      <div className="pb-4">{t("createAccount.alreadyHaveAccount")}</div>
+      <Button className="uppercase" type="button" onClick={signInRedirect}>
+        {t("label.signIn")}
+      </Button>
+    </Card.Section>
+  )
+}
+
+const onSubmit = () => {
+  console.log("hello")
+}
```

Comment:
For this, how will we get the form values to this submit function?

Reply:
Relatedly, how will errors be propagated up to the `CreateAccount` component? 

Would it be better to use one large "useForm" hook instead of making multiple mini forms?

Reply:
Yes I didn't think this through - thank you! Made that update, so lmk if that looks better